rspamd/entrypoint.sh

```diff
@@ -19,7 +19,12 @@ if [ $# -eq 0 ]; then
         mkdir -v -m 0750 "${dkim_dir}"
         echo "Generating DKIM signing key. Add the following TXT record to DNS domains:"
         rspamadm dkim_keygen -s "${RSPAMD_dkim_selector:?}" -b 2048 -k "${dkim_dir}/${RSPAMD_dkim_selector}.key" | tee "${dkim_dir}/${RSPAMD_dkim_selector}.txt"
-        chgrp -cR rspamd "${dkim_dir}"
+        chown -cR rspamd:rspamd "${dkim_dir}"
+        chmod -c g+r "${dkim_dir}"/*.key
+    # ensure correct ownership for DKIM keys (was missing in previous versions)
+    elif [ "$(stat -c '%U' "${dkim_dir}")" != "rspamd" ]; then
+        chown -cR rspamd:rspamd "${dkim_dir}"
+        chmod -c g+r "${dkim_dir}"/*.key
```

Comment:
It is possible but you fix only new installation because we[ test if the dir is not true](https://github.com/NethServer/ns8-mail/blob/e4b3e05ff66ad4e06868993ac2478c7db1983f29/rspamd/entrypoint.sh#L18), so for former installation we do not fix it.
so:
- test the permissions and set it accordingly only one time (what I did)
- or set it anyway all the time

Reply:
it is a regression, this is the evidences

```
[mail1@R1 state]$ cat environment 
....
IMAGE_URL=ghcr.io/nethserver/mail:1.4.2
MAIL_CLAMAV_IMAGE=ghcr.io/nethserver/mail-clamav:1.4.2
MAIL_DOVECOT_IMAGE=ghcr.io/nethserver/mail-dovecot:1.4.2
MAIL_HOSTNAME=R1.rocky9.org
MAIL_POSTFIX_IMAGE=ghcr.io/nethserver/mail-postfix:1.4.2
MAIL_RSPAMD_IMAGE=ghcr.io/nethserver/mail-rspamd:1.4.2
MODULE_ID=mail1
.....
```
[mail1@R1 state]$ podman exec -ti rspamd ash
/ # ls -ls /var/lib/rspamd/dkim/
total 8
     4 -rw-r-----    1 root     rspamd        1704 Jul  4 07:37 default.key
     4 -rw-r--r--    1 root     rspamd         454 Jul  4 07:37 default.txt
/ # ls -ls /var/lib/rspamd/
....
     0 drwxr-x---    2 root     rspamd          44 Jul  4 07:37 dkim
```